server/controllers/finance/reports/debtors/annual-clients-report.handlebars

```diff
@@ -30,10 +36,10 @@
             <tr>
               <td>{{row.accountNumber}}</td>
               <td>{{row.groupName }}</td>
-              <td class="text-right">{{currency row.openingBalance ../metadata.enterprise.currency_id}}</td>
-              <td class="text-right">{{currency row.debit  ../metadata.enterprise.currency_id}}</td>
-              <td class="text-right">{{currency row.credit  ../metadata.enterprise.currency_id}}</td>
-              <td class="text-right">{{currency row.closingBalance ../metadata.enterprise.currency_id}}</td>
+              <td class="text-right">{{debcred row.openingBalance row.currencyId}}</td>
+              <td class="text-right">{{debcred row.debit row.currencyId}}</td>
+              <td class="text-right">{{debcred row.credit row.currencyId}}</td>
+              <td class="text-right">{{debcred row.closingBalance row.currencyId}}</td>
```

Comment:
Is it possible to have also the movement column here, just after the credit column : 
```html
<td class="text-right">{{debcred row.movement  row.currencyId}}</td>
```